scripts/test_context_broker.sh

```diff
@@ -0,0 +1,243 @@
+#!/bin/bash -e
+
+echo "Obtaining token from Keycloak..."
+
+token=`curl -d "client_id=client1&grant_type=password&username=admin&password=admin" -X POST 'http://localhost:8085/auth/realms/master/protocol/openid-connect/token' | \
+python3 -c "import sys, json; print(json.load(sys.stdin)['access_token'])"`
```

Comment:
maybe better using jq than python to get the token? this way people who are just testing will not have to install full python

e.g.:

```
token=$( curl -sS --location --request POST 'http://localhost:8080/auth/realms/master/protocol/openid-connect/token' \
--header 'Content-Type: application/x-www-form-urlencoded' \
--data-urlencode 'username=user2' \
--data-urlencode 'password=user2' \
--data-urlencode 'grant_type=password' \
--data-urlencode 'client_id=client1' | jq -r ".access_token")
```

Reply:
is this really solved?